src/Connection.php

```diff
@@ -1077,7 +1077,7 @@ public function getNestTransactionsWithSavepoints(): bool
      *
      * @return mixed A string with the savepoint name or false.
      */
-    protected function _getNestedTransactionSavePointName()
+    protected function _getNestedTransactionSavePointName(): mixed
```

Comment:
The phpDoc is inaccurate, should be `string|false`. Or even better `?string`.

Reply:
Looking at the implementation and all the places where we call this method, I'd say our code will blow up if the method returns anything but a string.